src/CommunityToolkit.Maui.Core/Views/DrawingView/IDrawingView.shared.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.ObjectModel;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The DrawingView allows you to draw one or multiple lines on a canvas
+/// </summary>
+public interface IDrawingView : IView
+{
+	/// <summary>
+	/// Event occurred when drawing line completed
+	/// </summary>
+	/// <param name="lastDrawingLine">Last drawing line</param>
+	void DrawingLineCompleted(DrawingLine lastDrawingLine);
+
+	/// <summary>
+	/// The <see cref="Color"/> that is used by default to draw a line on the <see cref="IDrawingView"/>. This is a bindable property.
+	/// </summary>
+	Color LineColor { get; }
```

Comment:
What do you think about the name? Do we need Default prefix - DefaultLineColor?

Reply:
I would remove `This is a bindable property` since this is on `Core`.

Reply:
> What do you think about the name? Do we need Default prefix - DefaultLineColor?

No, because that will be the color line, and that can be changed on the `Control` implementation.